frame/composable-tests-helpers/src/test/proptest.rs

```diff
@@ -36,3 +36,36 @@ macro_rules! prop_assert_acceptable_computation_error {
 		);
 	}};
 }
+
+/// Accepts -2, -1, 0, 1, 2
+#[macro_export]
+macro_rules! prop_assert_zero_epsilon {
```

Comment:
very unclear naming

Reply:
proper_assert_around_zero(x, 2);

Reply:
@dzmitry-lahoda proper_assert_around_zero() is also not very clear :sweat_smile:  . And comment above definition helps understanding the expected behavior.